### ISSUE TITLE
NVDA announces incorrect list count for "File" list – says "1 of 9" instead of "1 of 6"

### ISSUE THREAD

**Issue by brianrob** [microsoft/perfview#2256]
Environment Details:
Application Name: PerfView
PerfView Version:3.1.21
Operating system: Windows 11 Enterprise (OS Build26100.4061)

Repro Steps:
Open PrefView.
Tab till left Navigation section.
Trun on the Narrator.
Observe that NVDA announces incorrect list count for "File" list – says "1 of 9" instead of "1 of 6"

Actual Result:
When navigating the "File" list using NVDA, it announces "1 of 9", even though only 6 items are visually and programmatically present in the list.

Expected Result:
NVDA should announce the correct number of items in the list, such as "1 of 6", matching the actual number of list elements.

User Impact:
Screen reader users receive misleading information about the number of items in the list, which can cause confusion, misnavigation, and reduce trust in the interface’s accessibility.

Additional Context:

Here is a screenshot of the issue:

<img width="375" height="355" alt="Image" src="https://github.com/user-attachments/assets/59d3504c-0443-4208-97ab-acf3d8376980" />

When the user has narrator enabled and they select one of the items in the File drop down menu, narrator will state that there are a total of 9 items in the drop down menu, when in fact there are only 6.  Narrator gets the number correct for the rest of the drop down menus.

**Comment by brianrob** [comment]
Fixed via #2257.
